.storybook/main.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  stories: ['../src/components/**/*.stories.@(ts|tsx)'],
+  managerEntries: ['@storybook/addon-links/register'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
```

Comment:
Maybe it would be good to add a config for docs for typescript as well: https://storybook.js.org/docs/react/configure/typescript
Unless it works fine without it now